src/main/java/me/totalfreedom/totalfreedommod/command/Command_unlinkreddit.java

```diff
@@ -20,7 +20,21 @@ public boolean run(CommandSender sender, Player playerSender, Command cmd, Strin
             msg("The reddit system is currently disabled.", ChatColor.RED);
             return true;
         }
-
+        // someone check this for me
```

Comment:
Could someone from @AtlasMediaGroup/totalfreedom-developers take a check over this, it _looks_ okay at a glance but my Java is rather rusty.